bigframes/session/clients.py

```diff
@@ -94,8 +96,22 @@ def __init__(
             else _APPLICATION_NAME
         )
         self._project = project
+
+        if (
+            use_regional_endpoints
+            and location is not None
+            and location.lower()
+            not in bigframes.constants.REP_ENABLED_BIGQUERY_LOCATIONS
+        ):
+            warnings.warn(
+                bigframes.constants.LEP_DEPRECATION_WARNING_MESSAGE.format(
+                    location=location
+                ),
+                category=bigframes.exceptions.DeprecationWarning,
```

Comment:
Let's use the built-in `FutureWarning`.

```suggestion
                category=FutureWarning,
```

Reply:
Done, thanks!